scripts/packages/fedora/bazel.spec

```diff
@@ -0,0 +1,43 @@
+Name: bazel
+Version: devel
+Release: 1
+Summary: Correct, reproducible, and fast builds for everyone.
+URL: https://bazel.build
+License: Apache License, v2.0
+
+Source0: bazel
+Source1: bazel-real
+Source2: bazel.bazelrc
+Source3: bazel-complete.bash
+
+Requires: java-1.8.0-openjdk-headless
```

Comment:
Not really. There's no package without the Java version. The -headless package is missing all video and audio code (which Bazel doesn't need) so it's smaller and has fewer dependencies.

Anyone trying to actually build will need the -devel package, which also has javac, but that's not needed just to install.